applications/ParticleMechanicsApplication/python_scripts/kratos_main_particle.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
 
 import KratosMultiphysics
-import KratosMultiphysics.ParticleMechanicsApplication
+import KratosMultiphysics.ParticleMechanicsApplication as KratosParticle
 from particle_mechanics_analysis import ParticleMechanicsAnalysis
```

Comment:
```suggestion
 from KratosMultiphysics.ParticleMechanicsApplication.particle_mechanics_analysis import ParticleMechanicsAnalysis
```
this is what became possible now with the python-modules